templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -160,12 +192,28 @@ Resources:
   VPCStack:
     Type: AWS::CloudFormation::Stack
     Properties:
-      TemplateURL: !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-aws-vpc/templates/aws-vpc.template
-        - QSS3Region: !If
-            - GovCloudCondition
-            - s3-us-gov-west-1
-            - s3
+      TemplateURL: !If
```

Comment:
Consider in code comment, a PR comment. It will be removed once we get agreement on why this logic is here. Nasty stuff, although it should make us backward compatible and prevent existing customers out there with their own buckets running into issues